### PR TITLE
Add information about scroll coordinates

### PR DESCRIPTION
**Pull request by PierreRambaud**
### Description of the feature or fix

While I was checking the documentation about "how I can restore the scroll of my items list", I didn't find anything in the documentation, I had to check the source code in https://github.com/lvgl/lvgl/blob/master/src/core/lv_obj_scroll.c in order to find a way to retrieve the current scroll data.

### Checkpoints
- [x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [x] Run `code-format.py` from the `scripts` folder. [astyle](http://astyle.sourceforge.net/install.html) needs to be installed.
- [x] Update the documentation

### CODE REVIEW

**Comment by PierreRambaud** [comment]
I don't get why tests are failing :thinking:

**Comment by embeddedt** [comment]
I agree that more documentation is needed here, but the current descriptions don't make much more sense to me than the code did, unfortunately.

Perhaps it's better to write a couple sentences beforehand explaining that `scroll_top`, for example, means the distance the object has been scrolled from the top (if that's correct).

(The CI failure is unrelated.)

**Comment by PierreRambaud** [comment]
> I agree that more documentation is needed here, but the current descriptions don't make much more sense to me than the code did, unfortunately.
> 
> Perhaps it's better to write a couple sentences beforehand explaining that `scroll_top`, for example, means the distance the object has been scrolled from the top (if that's correct).
> 
> (The CI failure is unrelated.)

I'm sorry, I'm not really good at writing documentation, I hope it's better right now!

File: docs/overview/scroll.md [hunk 1]
```diff
@@ -141,6 +141,28 @@ The following API functions allow manual scrolling of objects:
...
 - `lv_obj_scroll_to_y(obj, y, LV_ANIM_ON/OFF)` scroll to bring the given coordinate to the top side
 
+From time to time you may need to retrieve the scroll position of an element, either to restore it later, or to display dynamically some elements according to the current scroll.
+Here is an example to see how to combine scroll event and store the scroll top position.
+```c
+static int scroll_value = 0;
+
+static void store_scroll_value_event_cb(lv_event_t* e) {
+  lv_obj_t* screen = lv_event_get_target(e);
+  scroll_value = lv_obj_get_scroll_top(screen);
```
Comment (kisvegabor) [inline docs/overview/scroll.md:151]:
```suggestion
  scroll_value = lv_obj_get_scroll_top(screen);
  printf("%d pixels are scrolled out on the top\n", scroll_value);
```

**Review by kisvegabor — COMMENTED** [review]
It looks god to me. I've just added a minor suggestions.

File: docs/overview/scroll.md (hunk 1 above)
Comment (PierreRambaud) [reply docs/overview/scroll.md:151]: Thank you :pray:

**Comment by kisvegabor** [comment]
Thank you, merging! :slightly_smiling_face:

**Comment by kisvegabor** [comment]
I was smart enough to forget to press the merge button. :sweat_smile: 
Thanks!

**Pull request merged**